R/define_data.R

```diff
@@ -216,6 +216,135 @@ defDataAdd <- function(dtDefs = NULL,
   return(defNew[])
 }
 
+#' Add multiple (similar) rows to definitions table
+#'
+#' @param dtDefs Definition data.table to be modified
+#' @param nvars Number of new variables to define
```

Comment:
Ill rename this to nVars to be consistent with camelCase

Reply:
Sounds good.